### PR TITLE
Disable loading configuration from `.babelrc`.

### LINKED ISSUE #92: .babelrc in a parent directory causes build failure
I have a .babelrc file in the parent folder of my ember app which causes `ember serve` to fail. If I remove this file then the build works as expected. The file has the following contents:

``` js
{
  "presets": ["es2015-node4"]
}
```

The error that is generated is as follows:

```
File: modules/ember-inflector/index.js
The Broccoli Plugin: [Babel] failed with:
ReferenceError: [BABEL] modules/ember-inflector/index.js: Unknown option: /Users/simonwade/dev/Workspace/activistic-backend/.babelrc.presets
    at Logger.error (/Users/simonwade/dev/Workspace/activistic-backend/web-donation-form/node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/lib/transformation/file/logger.js:58:11)
    at OptionManager.mergeOptions (/Users/simonwade/dev/Workspace/activistic-backend/web-donation-form/node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/lib/transformation/file/options/option-manager.js:126:29)
    at OptionManager.addConfig (/Users/simonwade/dev/Workspace/activistic-backend/web-donation-form/node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/lib/transformation/file/options/option-manager.js:107:10)
    at OptionManager.findConfigs (/Users/simonwade/dev/Workspace/activistic-backend/web-donation-form/node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/lib/transformation/file/options/option-manager.js:168:35)
    at OptionManager.init (/Users/simonwade/dev/Workspace/activistic-backend/web-donation-form/node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/lib/transformation/file/options/option-manager.js:229:12)
    at File.initOptions (/Users/simonwade/dev/Workspace/activistic-backend/web-donation-form/node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/lib/transformation/file/index.js:147:75)
    at new File (/Users/simonwade/dev/Workspace/activistic-backend/web-donation-form/node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/lib/transformation/file/index.js:137:22)
    at Pipeline.transform (/Users/simonwade/dev/Workspace/activistic-backend/web-donation-form/node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/babel-core/lib/transformation/pipeline.js:164:16)
    at Babel.transform (/Users/simonwade/dev/Workspace/activistic-backend/web-donation-form/node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/index.js:105:21)
    at Babel.processString (/Users/simonwade/dev/Workspace/activistic-backend/web-donation-form/node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/index.js:204:25)

The broccoli plugin was instantiated at:
    at Babel.Plugin (/Users/simonwade/dev/Workspace/activistic-backend/web-donation-form/node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/broccoli-persistent-filter/node_modules/broccoli-plugin/index.js:7:31)
    at Babel.Filter [as constructor] (/Users/simonwade/dev/Workspace/activistic-backend/web-donation-form/node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/broccoli-persistent-filter/index.js:60:10)
    at new Babel (/Users/simonwade/dev/Workspace/activistic-backend/web-donation-form/node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/index.js:35:10)
    at Babel (/Users/simonwade/dev/Workspace/activistic-backend/web-donation-form/node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/index.js:30:12)
    at Object.registry.add.toTree (/Users/simonwade/dev/Workspace/activistic-backend/web-donation-form/node_modules/ember-cli-babel/index.js:23:52)
    at /Users/simonwade/dev/Workspace/activistic-backend/web-donation-form/node_modules/ember-cli/node_modules/ember-cli-preprocess-registry/preprocessors.js:184:26
    at Array.forEach (native)
    at processPlugins (/Users/simonwade/dev/Workspace/activistic-backend/web-donation-form/node_modules/ember-cli/node_modules/ember-cli-preprocess-registry/preprocessors.js:182:11)
    at Function.module.exports.preprocessJs (/Users/simonwade/dev/Workspace/activistic-backend/web-donation-form/node_modules/ember-cli/node_modules/ember-cli-preprocess-registry/preprocessors.js:175:10)
    at CoreObject.extend.preprocessJs (/Users/simonwade/dev/Workspace/activistic-backend/web-donation-form/node_modules/ember-cli/lib/models/addon.js:818:25)
```

### PR DESCRIPTION
**Pull request by aheuermann**
Fixes #92

### CODE REVIEW

**Comment by rwjblue** [comment]
Ok, just landed a PR that should have fixed CI, can you rebase this so we can get things green before landing?

**Comment by aheuermann** [comment]
Updated. Appveyor tests are still failing for node 4. I assume it is unrelated.

**Comment by rwjblue** [comment]
Yes, the failure seems to be during the `npm install` phase, and therefore pretty unlikely.

**Comment by rwjblue** [comment]
This change will disable `ember-cli-babel` from attempting to load configuration from `.babelrc` and `package.json`. That functionality was never intended to be enabled (I didn't realize this flag had to be specific disabled), but has worked for the duration of ember-cli-babel@6 so far. A number of folks have opened issues related to this and it is tricky to even know this is going on, but technically it _could_ be considered a breaking change.  I personally would categorize it as a "breaking bugfix", and think we should land it.

@stefanpenner / @Turbo87 - I'm 👍  on this, but want y'all to be aware and confirm that you agree with this change also before landing.

**Comment by aheuermann** [comment]
@Turbo87 @rwjblue I added a line to the readme.

File: README.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ target browsers (as specified in `config/targets.js` in ember-cli >= 2.13). Runn
 through the transpiler shouldn't change the code at all (likely just a format change if it does).
 
 If you need to customize the way that `babel-preset-env` configures the plugins that transform your code,
-you can do it by passing in any of the options found [here](https://github.com/babel/babel-preset-env#options).
+you can do it by passing in any of the options found [here](https://github.com/babel/babel-preset-env#options). 
+*Note: `.bablerc` files are ignored by default.*
```
Comment (rwjblue) [inline README.md]: Small typo here `s/bablerc/babelrc/`.

Comment (aheuermann) [inline README.md]: Fixed

**Comment by rwjblue** [comment]
Rebased to kick off another round of CI.

**Pull request merged**